tests/generate-tests-for-bad-variable-names.R

```diff
@@ -0,0 +1,88 @@
+### Title:    Generate Unit Tests for Data with Potentially Problematic Variable Names
+### Author:   Kyle M. Lang (Adapted from ./generate-tests.R)
+### Created:  2024-03-22
+### Modified: 2024-03-23
+
+###-Setup--------------------------------------------------------------------------------------------------------------------###
+
+## Assume our working directory is originally set to the modules project directory:
+setwd("tests/testthat")
+
+## Initialize the necessary subroutines:
+source("../test-generator-subroutines.R")
```

Comment:
I tried the `testthat::test_path()` solution, but I couldn't get it working. From what I can tell, the working directory needs to be set to the same place when building and running the tests, since I'm using a custom dataset that **jaspTools** doesn't know about. The data isn't found when running the tests if I generate the tests while using `test_path()` to resolve the paths.

I'm sure there's a more elegant solution, but I'm leaving the `setwd()` nonsense, for now.